### PR TITLE
Printfix2

### PR DESCRIPTION
**Pull request by emreyamangil**
This PR predefines dcp_objval and conic_objval to make sure their scope is ok, also adds a printing test.

### CODE REVIEW

File: test/conictest.jl [hunk 1]
```diff
@@ -116,4 +116,31 @@ end
...
+
+            problem = Convex.minimize(-3x-y,
+                               x >= 1,
+                               y >= 0,
+                               3x + 2y <= 10,
+                               x^2 <= 5,
+                               exp(y) + x <= 7)
+
+
+            Convex.solve!(problem, PajaritoSolver(verbose=1,algorithm=algorithm,mip_solver=mip_solver,cont_solver=conic_solver))
```
Comment (mlubin) [inline test/conictest.jl]: How does this test printing? We aren't checking the output

Comment (emreyamangil) [inline test/conictest.jl]: Just checking if we are crashing or not :) Should we test something else??

Comment (mlubin) [inline test/conictest.jl]: We could maybe capture the output and compare it against a known correct output

Comment (emreyamangil) [inline test/conictest.jl]: Will work on it as soon as I am in the office, thanks!!

File: src/conic.jl [hunk 2]
```diff
@@ -817,6 +817,9 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
             mip_solution = getValue(m.mip_x)
         end
 
+        dcp_objval = Inf
```
Comment (mlubin) [inline src/conic.jl]: What's the difference between these two variables?

Comment (emreyamangil) [inline src/conic.jl]: Lazy coding! :) Removed!!

**Comment by emreyamangil** [comment]
Sadly when we redirect_stdout, it gives pipe broken, will try to find a better way in the afternoon, for now added some cosmetic output changes only

**Pull request merged**